livingdoc-engine/src/main/kotlin/org/livingdoc/engine/fixtures/FixtureMethodInvoker.kt

```diff
@@ -116,10 +116,10 @@ class FixtureMethodInvoker(
     }
 
     class FixtureMethodInvocationException(method: Method, fixture: Any, e: Exception)
-        : RuntimeException("Could not invoke method '$method' on fixture '$fixture' because of an exception:", e)
+        : RuntimeException("Could not invoke method '${method.name}' on fixture '$fixture' because of an exception:", e)
```

Comment:
The name alone might not be enough to distinguish between two methods. I guess this should not be a problem in most cases, but the full signature is also the most helpful (?)